tui/tests/test_terminal_e2e.py

```
@@ -0,0 +1,1220 @@
+# ---------------------------------------------------------------------------
+# Calendar status line E2E — validates real Google Calendar data renders
+# ---------------------------------------------------------------------------
+
+
+class TestStatusLineCalendar:
+    """EXIT CONDITION: hookwise status-line shows calendar event data."""
+
+    @skipif_no_hookwise
+    def test_status_line_has_calendar_data(self):
+        """hookwise status-line output should contain calendar emoji or relative time."""
+        result = subprocess.run(
```

Ruff (0.15.4)

[error] `subprocess` call: check for execution of untrusted input

(S603)